honeybee_grasshopper_radiance/src/HB Annual Daylight.py

```diff
@@ -42,7 +38,7 @@
 
 ghenv.Component.Name = 'HB Annual Daylight'
 ghenv.Component.NickName = 'AnnualDaylight'
-ghenv.Component.Message = '1.1.0'
+ghenv.Component.Message = '1.2.0'
```

Comment:
FYI, for future reference, I think we should only increment the middle number when we are going to do a Food4Rhino update.  As it is now, this will create confusion if I go to release LBT Grasshopper version 1.2.0 and users don't see the version on all of the components change.